packages/avro-ts/src/types/enum.ts

```diff
@@ -1,20 +1,36 @@
 import { schema, Schema } from 'avsc';
 import { Convert } from '../types';
-import { Type } from '@ovotech/ts-compose';
+import { Node, Type } from '@ovotech/ts-compose';
 import { convertName, namedType, firstUpperCase } from '../helpers';
+import { withDefault } from './record';
 
-export const isEnumType = (type: Schema): type is schema.EnumType =>
+interface EnumWithDefault extends schema.EnumType {
+  default?: string,
```

Comment:
I have a [pull request](https://github.com/mtth/avsc/pull/402) to add this in `avsc`, but until that's accepted this was the short term fix.